lib/active_remote/integration.rb

```diff
@@ -63,7 +63,7 @@ def cache_key
       case
       when new_record? then
         "#{model_name.cache_key}/new"
-      when ::ActiveRemote.config.default_cache_key_updated_at? && (timestamp = self[:updated_at]) then
+      when ::ActiveRemote.config.default_cache_key_updated_at? && (timestamp = self.updated_at) then
```

Comment:
Will this generate a method missing error if `updated_at` is not a defined on the active remote record?

Reply:
I think it might... but is that a scenario we need to worry about?  It's pretty conventional to have `updated_at` as an attribute, and it will always have a method defined thanks to ActiveModel.

We could wrap the call in a `.try` though, or call it with `self["updated_at"]` and it should work. I am still not sure why this broke in the first place though. Very strange.

Reply:
While it's a common convention, it's a breaking change to the library since this will now result in an undefined method error. I would prefer hash accessor over. The fact that this broke is very curious indeed. @liveh2o any thoughts?

Reply:
I made the suggested change @film42 and hopefully handled any potential errors in a good way. Let me know what you think.